frontend/src/service/LocalStorage.js

```diff
@@ -0,0 +1,29 @@
+const ACCESS_TOKEN = 'ACCESS_TOKEN';
+const USER_DATA = 'USER_DATA';
+
+export const loadTokenFromLocalStorage = () =>
+  localStorage.getItem(ACCESS_TOKEN);
+
+export const saveTokenToLocalStorage = (token) =>
+  localStorage.setItem(ACCESS_TOKEN, token);
```

Comment:
Although we did it in the capstone manager, please understand the security implications of saving access tokens to local storage before adopting it as a general practice. See most voted answer here https://stackoverflow.com/questions/44133536/is-it-safe-to-store-a-jwt-in-localstorage-with-reactjs